main.go

```diff
@@ -1543,6 +1570,8 @@ func main() {
 		Monitor:         *monitor,
 		BaudRate:        *baudrate,
 		Timeout:         *timeout,
+		WitPackage:      witPackage,
+		WitWorld:        witWorld,
```

Comment:
Should probably rename these to WITPackage and WITWorld consistent with Go initialisms convention.

(Elsewhere too)

Reply:
Done.